internal/service/identity_api/request.go

```diff
@@ -0,0 +1,31 @@
+package identity_api
```

Comment:
I can't roll with an underscore in a package name. I think `identity` is a fine package name for this, in any case.